terraform/environments/digital-prison-reporting/data_product_definitions.tf

```diff
@@ -0,0 +1,132 @@
+locals {
+  table_name = "${local.project}-data-product-definition"
+}
+
+module "dynamo_table_dpd" {
+  source              = "./modules/dynamo_tables"
+  create_table        = true
+  autoscaling_enabled = false
+  name                = local.table_name
+
+  hash_key    = "data-product-id"
+  range_key   = "category"
+  table_class = "STANDARD"
+  ttl_enabled = false
+
+  attributes = [
+    {
+      name = "data-product-id"
+      type = "S"
+    },
+    {
+      name = "category"
```

Comment:
What is category?

Reply:
I was trying to think of a better name than "path", given there is no folder path anymore.

Happy to carry on using path, though.